en/components/grid.md

```diff
@@ -61,11 +61,9 @@ Now that we have the grid module imported, let’s get started with a basic conf
 <igx-grid #grid1 id="grid1" [data]="localData" [autoGenerate]="true"></igx-grid>
 ```
 
-The **id** property is a string value and is the unique identifier of the grid, while **data** binds the grid, in this case to local data.
-
-The **autoGenerate** property tells the **igx-grid** to autogenerate columns based on the data source fields. Otherwise, the developer needs to explicitly define the columns and the mapping to the data source fields.
+The **id** property is a string value and is the unique identifier of the grid which will be autogenerated if not provided, while **data** binds the grid, in this case to local data.
 
-<div class="divider--half"></div>
+The **autoGenerate** property tells the **igx-grid** to autogenerate columns based on the data source fields. It will also try to deduce the appropriate data type for the column if possible. Otherwise, the developer needs to explicitly define the columns and the mapping to the data source fields.
```

Comment:
Refer to columns as column objects and their type. "to auto generate the grid's `IgxColumnComponent` objects...`.